sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/expected/com/sample/TestModel.java

```diff
@@ -0,0 +1,129 @@
+package com.sample;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import com.squareup.sqldelight.ColumnAdapter;
+import com.squareup.sqldelight.RowMapper;
+import java.lang.Boolean;
+import java.lang.Override;
+import java.lang.String;
+import java.util.List;
+
+public interface TestModel {
+  String TABLE_NAME = "test";
+
+  String ASC = "'ASC'";
+
+  String DESC = "\"DESC\"";
+
+  String TEXT = "`TEXT`";
+
+  String BOOLEAN = "[Boolean]";
```

Comment:
no idea. we should certainly keep it here because sqlite will expect them in this form. whether or not `cursor.getColumnIndex` expects the quoted version or not is what i'll have to check but most likely it does use quotes.